docs/runbooks/LowReadyVirtControllersCount.md

```diff
@@ -2,42 +2,44 @@
 
 ## Meaning
 
-From a high-level perspective the virt-controller has all the cluster wide virtualization functionality.
+This alert fires when one or more `virt-controller` pods are running, but none of these pods have been in a `Ready` state for the last 5 minutes. 
```

Comment:
Small English usage comment: "one or more" and "none of ..." use singular verbs.

This alert fires when one or more ... is running, but none of the pods has been ....

Reply:
OK, leave it as it is.